### PR TITLE
Merge installer and upgrader packages

### LINKED ISSUE #833: cloud-config file was not changed during cluster upgrade
**What happened**:
After upgrading a GCP based k1-cluster from 1.15.5 to 1.15.11 (and later  1.16.8) via `kubeone upgrade config.yaml --tfjson tf.json` the cloud-controller could not create routes for (worker?) nodes anymore.
It always errored with: `Warning  FailedToCreateRoute      2m37s                 route_controller                                Could not create route 567b09ad-db20-46c5-aec0-8f19d6f814e3 10.244.10.0/24 for node run-2-alpha-pool1-77bc85f967-l6hpq after 218.408669ms: instance not found`
After a while of fiddling around, I found out, that the cloud-config at /etc/kubernetes/cloud-config was empty and the kubeone upgrade command (I appended `--force` to retry) did NOT copy the expected values from the cluster configuration:

```yaml
apiVersion: kubeone.io/v1alpha1
kind: KubeOneCluster
name: run-2-alpha
versions:
  kubernetes: '1.16.8'
cloudProvider:
  name: 'gce'
  cloudConfig: |
    [global]
    multizone = true
```

I manually sshed into the master-nodes, issued a

```shell
cat > /etc/kubernetes/cloud-config <<EOF
[global]
multizone = true
EOF
```

and rebootet them one-by-one, which fixed the issue.

**What is the expected behavior**: Kubeone should update the cloud-config file content based on config yaml input.

**How to reproduce the issue**: ?

**Anything else we need to know?**

**Information about the environment**:
KubeOne version (`kubeone version`): 0.11.0
Operating system: Ubuntu 19.10
Provider you're deploying cluster on: GCP
Operating system you're deploying on: Ubuntu 18.04.3 LTS

### PR DESCRIPTION
**Pull request by kron4eg**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #833

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
Fixes cloud-config file was not changed during cluster upgrade #833
```

### CODE REVIEW

**Comment by xmudrii** [comment]
/retest
Seems like AWS was flaky last night

File: pkg/tasks/controlplane.go [hunk 1]
```diff
@@ -25,14 +25,36 @@ import (
 	"github.com/kubermatic/kubeone/pkg/state"
 )
 
+func joinControlplaneNode(s *state.State) error {
+	s.Logger.Infoln("Joining controlplane node…")
+	return s.RunTaskOnFollowers(joinControlPlaneNodeInternal, state.RunSequentially)
```
Comment (xmudrii) [inline pkg/tasks/controlplane.go:30]: At some point, we should consider running this parallel.

File: pkg/tasks/machine_deployments.go [hunk 2]
```diff
@@ -14,22 +14,30 @@ See the License for the specific language governing permissions and
...
 import (
 	"context"
 
-	"github.com/pkg/errors"
-
-	"github.com/kubermatic/kubeone/pkg/state"
-
 	clusterv1alpha1 "github.com/kubermatic/machine-controller/pkg/apis/cluster/v1alpha1"
-
+	"github.com/pkg/errors"
```
Comment (xmudrii) [inline pkg/tasks/machine_deployments.go:23]: This should go in a group above.

File: pkg/tasks/task.go [hunk 3]
```diff
@@ -41,22 +41,23 @@ type Task struct {
...
-		lastError = t.Fn(ctx)
+		lastError = t.Fn(s)
 		if lastError != nil {
-			ctx.Logger.Warn("Task failed…")
-			if ctx.Verbose {
-				ctx.Logger.Warnf("error was: %s", lastError)
+			s.Logger.Warn("Task failed…")
+			if s.Verbose {
+				s.Logger.Warnf("error was: %s", lastError)
 			}
```
Comment (xmudrii) [inline pkg/tasks/task.go:61]:
```suggestion
			s.Logger.Warnf("error was: %s", lastError)
```

I propose that we just print the error when the task fails regardless of is verbose enabled or not. In some cases, such as `kubeadm init` or `kubeadm join`, the error can be different between retries and that can be confusing. Printing error each time it happens should make the user experience a little bit better.

Comment (kron4eg) [reply pkg/tasks/task.go:61]: 👍 sounds good

**Pull request merged**